### ISSUE TITLE
Does pyogrio automatically reproject the geodataframe into WGS84/EPSG:4326?

### ISSUE THREAD

**Issue by ZeroCool2u** [geopandas/pyogrio#143]
The geojson spec seems to suggest that all geojson is expected to be WGS84/EPSG:4326. 

It's fine if pyogrio does not reproject the data before writing, but I don't see any mention of this in the docs and want to understand if I'm assuming incorrectly. 

Thanks!

**Comment by brendan-ward** [comment]
Data are [automatically reprojected by GDAL](https://gdal.org/drivers/vector/geojson.html) when using the GeoJSON driver; no reprojection is performed by pyogrio.

We could mention this in the docs.

**Comment by brendan-ward** [comment]
You're right, my mistake.  I was misled by QGIS which claimed it was in WGS84, when indeed, the coordinates are not reprojected by default.

To use the option to automatically reproject in GDAL:

```Python
write_dataframe(gdf, filename, RFC7946="YES")
```

**Comment by ZeroCool2u** [comment]
Thank you for the clarification! I just found out that the same issue is also present in GeoPandas, but I've started using this library and figured it would be good to document for others in the future either way.
